Avoid dysphemisms (#2383)
scala/docs.scala-lang: _overviews/contribute/inclusive-language-guide.md

You may want to use automated software like [In Solidarity](https://github.com/apps/in-solidarity) to steer contributors away from loaded words.

## Dysphemism

> Dysphemisms, the opposite of euphemisms, should be avoided. For example, the English expressions "pull the trigger" (enforce a decision) and "bite the bullet" (endure hardship) can be disturbingly violent if you are not used to them.
> 
> Using the direct meaning is preferred.

Reviewer: To follow the writing style of the rest of the document, this should read as: (also one Markdown line per sentence, for better diffs later on)
```suggestion
Dysphemisms, the opposite of euphemisms, can be disturbingly violent if you are not used to them.
Examples include the English expressions "pull the trigger" (enforce a decision) and "bite the bullet" (endure hardship).
Prefer the direct meaning instead.
```
i.e.:

1. Problem
2. Examples
3. Propose a solution

with as much active voice as possible.
